.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.formatting.provider": "yapf"
```

Comment:
I would prefer we do not commit IDE specific code here. Please add it in gitignore